package.json

```diff
@@ -1,27 +1,31 @@
 {
   "name": "@ensdomains/dnssec-oracle",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "requires": true,
   "lockfileVersion": 1,
   "devDependencies": {
     "@ensdomains/buffer": "^0.0.8",
     "@ensdomains/solsha1": "^0.0.2",
+    "@nomiclabs/hardhat-truffle5": "^2.0.0",
+    "@nomiclabs/hardhat-web3": "^2.0.0",
+    "@openzeppelin/test-helpers": "^0.5.10",
     "@truffle/hdwallet-provider": "^1.2.1",
     "dnsprovejs": "0.0.9",
```

Comment:
The latest dnsprovejs is 0.3.7

Reply:
Actually, it's `@ensdomains/dnsprovejs`.  You are more likely importing wrong package